app/frontend/components/domains/requirement-template/edit-requirement-template-screen/builder-header.tsx

```diff
@@ -63,9 +61,10 @@ export const BuilderHeader = observer(function BuilderHeader({ requirementTempla
         </HStack>
         <HStack>
           <TemplateStatusTag requirementTemplate={requirementTemplate} />
-          <Tag py={1} px={2} borderRadius="sm" backgroundColor={"greys.grey03"} color={"text.secondary"}>
-            {stubVersion}
-          </Tag>
+          {/*  TODO: versioning is changing, implement later*/}
+          {/*<Tag py={1} px={2} borderRadius="sm" backgroundColor={"greys.grey03"} color={"text.secondary"}>*/}
+          {/*  {stubVersion}*/}
+          {/*</Tag>*/}
```

Comment:
can this be completed now? E.g. `requirementTemplate.publishedTemplateVersion.versionDate`?

Reply:
@LaurelOlson this is handled in my latest PR thanks